tools/Replay.hpp

```diff
@@ -29,6 +29,20 @@
 
 #include <iostream>
 #include <fstream>
+#include <config_version.h>
+#include <ctime>
+#include <Util.hpp>
+#include <common/Log.hpp>
+
+
```

Comment:
Unusually large to have 3x spaces - vertical space is a non-renewable resource to be used tastefully =)